mne/decoding/tests/test_transformer.py

```diff
@@ -160,3 +160,45 @@ def test_epochs_vectorizer():
     # Test init exception
     assert_raises(ValueError, vector.fit, epochs, y)
     assert_raises(ValueError, vector.transform, epochs, y)
+
+
+def test_vectorzer():
+    """Test whether vectorizer is working"""
+
+    raw = io.read_raw_fif(raw_fname, preload=False)
+    events = read_events(event_name)
+    picks = pick_types(raw.info, meg=True, stim=False, ecg=False,
+                       eog=False, exclude='bads')
+    picks = picks[1:13:3]
+    with warnings.catch_warnings(record=True):
+        epochs = Epochs(raw, events, event_id, tmin, tmax, picks=picks,
+                        baseline=(None, 0), preload=True)
+    epochs_data = epochs.get_data()
+    vector = Vectorizer(epochs.info)
```

Comment:
maybe this should not be called vector. It should just be `vectorizer` or similar.